Sources/App/Views/Author/AuthorShow+View.swift

```diff
@@ -0,0 +1,39 @@
+import Plot
+
+
+enum AuthorShow {
+
+    class View: PublicPage {
+
+        let model: Model
+
+        init(path: String, model: Model) {
+            self.model = model
+            super.init(path: path)
+        }
+
+        override func content() -> Node<HTML.BodyContext> {
+            .div(
+                .class("author recent"), // 'recent' is temporary while I figure out how to regenerate the CSS file from SCSS
```

Comment:
Looks like you need CodeKit3 (paid app) in order to generate the CSS from SCSS? If someone else could give us a hand here I think the SCSS is right!